### PR TITLE
[spring-cloud-config] Initial integration for spring-cloud-config-client

### PR DESCRIPTION
**Pull request by aschaich**

### CODE REVIEW

**Comment by aschaich** [comment]
Same issue as in https://github.com/google/oss-fuzz/pull/8751#issuecomment-1279019429

File: projects/spring-cloud-config/src/main/java/ossfuzz/EnvironmentFuzzer.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	public EnvironmentFuzzer( FuzzedDataProvider fuzzedDataProvider) {
+		this.fuzzedDataProvider = fuzzedDataProvider;
+
+	}
+	
+	void test()
+	{
+		String[] fuzzedArray = {fuzzedDataProvider.consumeString(10), fuzzedDataProvider.consumeString(10)};
+		Environment environment = new Environment(fuzzedDataProvider.consumeString(10),
+												  fuzzedArray,
```
Comment (Alan32Liu) [inline projects/spring-cloud-config/src/main/java/ossfuzz/EnvironmentFuzzer.java:39]: The indentation seems to be a bit strange.

**Review by Alan32Liu — APPROVED** [review]
Approving with a minor comment.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**